Source/ConversationV1/Models/MessageResponse.swift

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+/** A conversation response. */
+public struct MessageResponse: JSONDecodable {
+    
+    // Context, or state, from the Conversation system
+    public var context:  [String: JSON]?
```

Comment:
I think `context` should be of type `JSON` instead of `[String: JSON]`. There isn't anything in the documentation specifying that context will be a dictionary with string keys, right? It would be odd for it to be anything else, but stranger things have happened. ;)